client/dashboard/me/mcp/setup/index.tsx

```diff
@@ -0,0 +1,411 @@
+		{ label: 'Claude Desktop', value: 'claude' },
+		{ label: 'Cursor', value: 'cursor' },
+		{ label: 'VS Code', value: 'vscode' },
+		{ label: 'Continue', value: 'continue' },
+		{ label: __( 'Other MCP client' ), value: 'default' },
+	];
+
+	// Documentation links for each client
+	const clientDocumentation: Record< McpClient, string > = {
+		claude: 'https://docs.claude.com/en/docs/mcp',
+		vscode: 'https://code.visualstudio.com/docs/copilot/customization/mcp-servers',
+		cursor: 'https://docs.cursor.com/en/context/mcp',
+		continue: 'https://docs.continue.dev/customize/deep-dives/mcp',
+		default: 'https://modelcontextprotocol.io/docs/develop/connect-local-servers',
+	};
+
+	const serverName = 'wpcom-mcp';
+
+	// Generate MCP configuration based on selected client
+	const generateMcpConfig = ( client: McpClient ) => {
+		const baseConfig = {
+			command: 'npx',
+			args: [ '-y', '@automattic/mcp-wpcom-remote@latest' ],
+		};
+
+		switch ( client ) {
+			case 'claude':
+				return {
+					mcpServers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+			case 'vscode':
+				return {
+					servers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+			case 'cursor':
+				return {
+					mcpServers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+			case 'continue':
+				return {
+					mcpServers: [
+						{
+							name: serverName,
+							...baseConfig,
+						},
+					],
+				};
+			default:
+				return {
+					mcpServers: {
+						[ serverName ]: baseConfig,
+					},
+				};
+		}
+	};
+
+	// Copy MCP configuration to clipboard
+	const copyToClipboard = async () => {
+		const configText = JSON.stringify( generateMcpConfig( selectedMcpClient ), null, 2 );
+
+		try {
+			await navigator.clipboard.writeText( configText );
+			setCopyStatus( 'success' );
+			setTimeout( () => setCopyStatus( 'idle' ), 2000 );
+		} catch ( err ) {
+			setCopyStatus( 'error' );
+			setTimeout( () => setCopyStatus( 'idle' ), 2000 );
+		}
+	};
+
+	// Helper function to get the appropriate icon based on copy status
+	const getCopyIcon = () => {
+		switch ( copyStatus ) {
+			case 'success':
+				return check;
+			case 'error':
+				return error;
+			default:
+				return copy;
+		}
+	};
+
+	// Check if any account-level tools are enabled using the new nested structure
+	const hasEnabledTools = hasEnabledAccountTools( userSettings || {} );
+
+	if ( ! hasEnabledTools ) {
+		return (
+			<PageLayout
+				size="small"
+				header={
+					<PageHeader
+						title={ __( 'MCP Client Setup' ) }
+						description={ __( 'Configure your MCP client to connect to WordPress.com.' ) }
+						prefix={ <Breadcrumbs length={ 2 } /> }
+					/>
+				}
+			>
+				<ComponentViewTracker eventName="calypso_dashboard_mcp_setup_view" />
+				<Card>
+					<CardBody>
+						<SectionHeader level={ 3 } title={ __( 'Setup Required' ) } />
+						<VStack spacing={ 4 }>
+							<Text as="p" size="medium" variant="muted">
```

Comment:
I think we should remove any usage of `size="medium"` as it doesn't match the styles on Figma